tutorials/README.rst

```diff
@@ -126,6 +126,18 @@ Notes on how to contribute more notebooks, and how the rendering is made, are at
      - `Scott Daniel <https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@danielsf>`_
 
 
+   * - Truth Catalog with GCR: variables and transients
+     - Example of accessing variables and transient objects in the truth catalog with GCR
+     - `ipynb <truth_gcr_variables.ipynb>`_,
+       `rendered <https://nbviewer.jupyter.org/github/LSSTDESC/DC2-analysis/blob/rendered/tutorials/truth_gcr_variables.nbconvert.ipynb>`_
+
+       .. image:: https://github.com/LSSTDESC/DC2-analysis/blob/rendered/tutorials/log/truth_gcr_variables.svg
+          :target: https://github.com/LSSTDESC/DC2-analysis/blob/rendered/tutorials/log/truth_gcr_variables.log
+
+     - `Yao-Yuan Mao <https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@yymao>`_, 
+       `Scott Daniel <https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@danielsf>`_
+
+
    * - Extragalactic catalog with GCR: redshift distributions
      - Extract, plot and explore the differential number counts of galaxies
      - `ipynb <extragalactic_gcr_redshift_dist.ipynb>`_,
```

Comment:
We want to encourage folks to modify/adjust their own ipynbs to do what they want, perhaps a nudge here would help, reminding folks that we don't mind if there is a reasonable amount of code repeated from the ipynbs in the submissions for HackurDC2